### ISSUE TITLE
Bench initialization fails on linux/arm64 (also on locally built image on M1 Macs)

### ISSUE THREAD

**Issue by danimelchor** [frappe/frappe_docker#595]
## Description of the issue
Error running bench init inside development container.

## Context information (for bug reports)
Newest M1 Pro MacBook.
Docker version: `20.10.8`
OS: `MacOS Monterrey Version 12.0.1`

## Steps to reproduce the issue

1. Follow the guide [here](https://github.com/frappe/frappe_docker/tree/main/development) until `"Setup first bench"`
2. Running `bench init --skip-redis-config-generation --frappe-branch version-13 frappe-bench` gives error

### Observed result
Error running bench init without a message.

### Expected result
Should correctly initialize the bench for frappe.

### Stacktrace / full error message if available

```bash
frappe@124b158d3d36:/workspace/development$ bench init --skip-redis-config-generation --frappe-branch version-13 frappe-bench
WARN: bench is installed in editable mode!

This is not the recommended mode of installation for production. Instead, install the package from PyPI with: `pip install frappe-bench`

$ /home/frappe/.local/bin/virtualenv --quiet env -p python3
        $ /workspace/development/frappe-bench/env/bin/python -m pip install --quiet --upgrade pip
cd frappe-bench
Getting frappe
$ git clone https://github.com/frappe/frappe.git --branch version-13 --depth 1 --origin upstream
Cloning into 'frappe'...
remote: Enumerating objects: 2949, done.
remote: Counting objects: 100% (2949/2949), done.
remote: Compressing objects: 100% (2772/2772), done.
remote: Total 2949 (delta 402), reused 1055 (delta 105), pack-reused 0
Receiving objects: 100% (2949/2949), 16.23 MiB | 3.07 MiB/s, done.
Resolving deltas: 100% (402/402), done.
Updating files: 100% (2817/2817), done.
Installing frappe
$ /workspace/development/frappe-bench/env/bin/python -m pip install --quiet --upgrade -e /workspace/development/frappe-bench/apps/frappe 
$ yarn install
yarn install v1.22.17
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning " > bootstrap@4.5.0" has unmet peer dependency "jquery@1.9.1 - 3".
warning " > bootstrap@4.5.0" has unmet peer dependency "popper.js@^1.16.0".
warning Workspaces can only be enabled in private projects.
[4/4] Building fresh packages...
[1/4] ⢀ node-sass
[2/4] ⢀ core-js
[3/4] ⢀ core-js
Traceback (most recent call last):
  File "/home/frappe/.bench/bench/commands/make.py", line 70, in init
    init(
  File "/home/frappe/.bench/bench/utils/render.py", line 95, in wrapper_fn
    return fn(*args, **kwargs)
  File "/home/frappe/.bench/bench/utils/system.py", line 78, in init
    get_app(
  File "/home/frappe/.bench/bench/app.py", line 355, in get_app
    app.install(verbose=verbose, skip_assets=skip_assets)
  File "/home/frappe/.bench/bench/utils/render.py", line 110, in wrapper_fn
    return fn(*args, **kwargs)
  File "/home/frappe/.bench/bench/app.py", line 186, in install
    install_app(
  File "/home/frappe/.bench/bench/app.py", line 397, in install_app
    bench.run("yarn install", cwd=app_path)
  File "/home/frappe/.bench/bench/bench.py", line 43, in run
    return exec_cmd(cmd, cwd=cwd or self.cwd)
  File "/home/frappe/.bench/bench/utils/__init__.py", line 126, in exec_cmd
    raise CommandFailedError
bench.exceptions.CommandFailedError

ERROR: There was a problem while creating frappe-bench
```

**Comment by revant** [comment]
I don't have a mac. I can't help much.

@vrslev @ddavidebor anyone of you have the machine?

**Comment by vrslev** [comment]
*Works fine on my machine!*
```console
> bench init --skip-redis-config-generation --frappe-branch version-13 frappe-bench --verbose
...
SUCCESS: Bench frappe-bench initialized
```
@danimelchor Can you try on latest frappe/bench image? Run `docker rmi frappe/bench -f; docker pull frappe/bench` and try again.

**Comment by vrslev** [comment]
@revant Also on the same topic. Dev Container crashes on my M1 Mac every ~20 sec. After building image locally, it does not.

**Comment by vrslev** [comment]
@danimelchor Did you build bench image yourself? Just tried entering Dev Container with locally built image and bench initialization fails! PhantomJS not found for linux/arm64 architecture:
```console
frappe@1114691b3eae:/workspace/development$ bench init --skip-redis-config-generation --frappe-branch version-13 frappe-bench
...
$ yarn install
yarn install v1.22.17
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning " > bootstrap@4.5.0" has unmet peer dependency "jquery@1.9.1 - 3".
warning " > bootstrap@4.5.0" has unmet peer dependency "popper.js@^1.16.0".
warning Workspaces can only be enabled in private projects.
[4/4] Building fresh packages...
[1/4] ⠈ node-sass
[-/4] ⠈ waiting...
[-/4] ⠈ waiting...
error /workspace/development/frappe-bench/apps/frappe/node_modules/phantomjs-prebuilt: Command failed.
Exit code: 1
Command: node install.js
Arguments: 
Directory: /workspace/development/frappe-bench/apps/frappe/node_modules/phantomjs-prebuilt
Output:
PhantomJS not found on PATH
Unexpected platform or architecture: linux/arm64



Traceback (most recent call last):
  File "/home/frappe/.bench/bench/commands/make.py", line 70, in init
    init(
  File "/home/frappe/.bench/bench/utils/render.py", line 95, in wrapper_fn
    return fn(*args, **kwargs)
  File "/home/frappe/.bench/bench/utils/system.py", line 78, in init
    get_app(
  File "/home/frappe/.bench/bench/app.py", line 355, in get_app
    app.install(verbose=verbose, skip_assets=skip_assets)
  File "/home/frappe/.bench/bench/utils/render.py", line 110, in wrapper_fn
    return fn(*args, **kwargs)
  File "/home/frappe/.bench/bench/app.py", line 186, in install
    install_app(
  File "/home/frappe/.bench/bench/app.py", line 412, in install_app
    bench.run("yarn install", cwd=app_path)
  File "/home/frappe/.bench/bench/bench.py", line 43, in run
    return exec_cmd(cmd, cwd=cwd or self.cwd)
  File "/home/frappe/.bench/bench/utils/__init__.py", line 126, in exec_cmd
    raise CommandFailedError
bench.exceptions.CommandFailedError

ERROR: There was a problem while creating frappe-bench
Do you want to rollback these changes? [y/N]: 
```

I will open issue in Frappe framework repo.

**Comment by vrslev** [comment]
https://github.com/frappe/frappe/issues/15192

**Comment by vrslev** [comment]
This issue was resolved in Frappe [v13.17.0](https://github.com/frappe/frappe/releases/tag/v13.17.0). Tested locally — everything works.
